src/tab-container/index.tsx

```diff
@@ -181,7 +181,7 @@ export const TabContainer = factory(function TabContainer({
 	];
 
 	let alignClass;
-	let orientation = 'horizontal';
+	let orientation: 'horizontal' | 'vertical' = 'horizontal';
```

Comment:
Should we create a type for this?

Reply:
I changed it to use the existing property on the `AriaAttributes` interface